app/_src/kubernetes-ingress-controller/get-started/rate-limiting.md

```diff
@@ -6,3 +6,78 @@ type: tutorial
 purpose: |
   How to configure rate limiting with Redis
 ---
+
+Rate limiting is used to control the rate of requests sent to an upstream service. It can be used to prevent DoS attacks, limit web scraping, and other forms of overuse. Without rate limiting, clients have unlimited access to your upstream services, which may negatively impact availability.
+
+{{site.base_gateway}} imposes rate limits on clients through the use of the [Rate Limiting plugin](/hub/kong-inc/rate-limiting/).  When rate limiting is enabled, clients are restricted in the number of requests that can be made in a configurable period of time.  The plugin supports identifying clients as consumers based on authentication or by the client IP address of the requests.
+
+## Create a rate-limiting KongPlugin
+
+Configuring plugins with {{ site.kic_product_name }} is different to how you'd do it with {{ kic.base_gateway }}. Rather than attaching a configuration directly to a service or route, you create a `KongPlugin` definition and then annotate your Kubernetes resource with the `konghq.com/plugins` annotation.
```

Comment:
```suggestion
Configuring plugins with {{ site.kic_product_name }} is different compared to how you'd do it with {{ kic.base_gateway }}. Rather than attaching a configuration directly to a service or route, you create a `KongPlugin` definition and then annotate your Kubernetes resource with the `konghq.com/plugins` annotation.
```